include/gatekeeper_lpm.h

```diff
@@ -19,10 +19,23 @@
 #ifndef _GATEKEEPER_LPM_H_
 #define _GATEKEEPER_LPM_H_
 
+#include <arpa/inet.h>
+
 #include <rte_lpm.h>
 #include <rte_lpm6.h>
 
 /* TODO Implement functions to edit IPv4/IPv6 routes. */
+struct ipv4_lpm_route {
+	uint32_t ip;
+	uint8_t  depth;
+	uint8_t  nexthop;
```

Comment:
`nexthop` cannot be `uint8_t`; otherwise, we won't support more than 256 fib entries. Check what DPDK is using and adopt the same. The same goes for the IPv6 case.